src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import ball.characteristics.Ball;
+import ball.characteristics.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

Reply:
not fixed - we should avoid static methods, this will become evident in Spring section